docs/problem_format/custom_graders.md

```diff
@@ -70,7 +70,8 @@ In this example, it's important to note the `unbuffered` node. If set to `true`,
 
 # Interactive Grading
 
-Interactive grading is used for problems where users should implement an online algorithm or where the grader must make generate input or compute a score based on the user's previous output. Using an interactive grader is similar to using a custom grader: the `custom_judge` node also needs to be set. Rewriting the previous custom judge using an interactive grader would result in:
+Interactive grading is used for problems where users should implement an online algorithm or where the grader must make generate input or compute a score based on the user's previous output.
+Using an interactive grader is similar to using a custom grader: the `custom_judge` node also needs to be set. Rewriting the previous custom judge using an interactive grader would result in:
```

Comment:
Let's take this opportunity to rename `input` in the following code to something that doesn't shadow `input`.